### PR TITLE
support @inheritdoc

### PR DESCRIPTION
**Pull request by jens1o**
### Description of the Change
Support the phpdoc keyword `@inheritDoc`

### Alternate Designs
It looked the best to me, and I looked after other things had been implemented.

### Benefits
`@inheritDoc` will be marked as a keyword, so the code will be displayed better.

### Possible Drawbacks
I don't know any.

### Applicable Issues
Did I need to create one?

### CODE REVIEW

File: grammars/php.cson [hunk 1]
```diff
@@ -1489,7 +1489,7 @@
         'match': '(?:(\\=)(&))|(&(?=[$A-Za-z_]))'
       }
       {
-        'match': '(@)'
+        'match': '(f)'
```
Comment (jens1o) [inline grammars/php.cson]: that wasn't by purpose...

**Comment by jens1o** [comment]
Okay, will do this. Thanks for the heads up!

**Comment by jens1o** [comment]
Question: Do I need to worry about uppercase? Usually it is `@inheritDoc`, but sometimes it's just `@inheritdoc`.

**Comment by Ingramz** [comment]
Oh, I am sorry, both are supported. Keep your old changes too :-)

File: grammars/php.cson [hunk 2]
```diff
@@ -1723,14 +1723,14 @@
         'match': '(@xlink)\\s+(.+)\\s*$'
       }
       {
-        'match': '\\@(a(pi|bstract|uthor)|c(ategory|opyright)|example|global|internal|li(cense|nk)|method|p(roperty(\\-read|\\-write|)|ackage|aram)|return|s(ee|ince|ource|tatic|ubpackage)|t(hrows|odo)|v(ar|ersion)|uses|deprecated|final|ignore)\\b'
+        'match': '\\@(a(pi|bstract|uthor)|c(ategory|opyright)|example|global|in(herit|ternal)|li(cense|nk)|method|p(roperty(\\-read|\\-write|)|ackage|aram)|return|s(ee|ince|ource|tatic|ubpackage)|t(hrows|odo)|v(ar|ersion)|uses|deprecated|final|ignore)\\b'
```
Comment (Ingramz) [inline grammars/php.cson]: Should it also be inheritdoc here? Looks like it will highlight `@inherit` at the moment.

Comment (jens1o) [inline grammars/php.cson]: Yeah, just a typo. Fixed. Thanks for the heads up anyway ;)

**Comment by Ingramz** [comment]
Last thing, does `@inheritDoc` with uppercase-D work?

**Comment by jens1o** [comment]
That's what I've been asked 😄 

This is my first pr in kind of language grammar, I don't know wether it is passed lowercase or just how it is.

**Comment by Ingramz** [comment]
Quick test shows that it does not work. Use `[Dd]` instead of `d` to allow both variants.

**Comment by jens1o** [comment]
Okay, thanks!

**Pull request merged**